_sass/components/_example-tabs.scss

```diff
@@ -102,7 +102,11 @@
           line-height: var(--line-height-sm);
 
           > a {
-            width: fit-content;
+            // FIXME: This rule is deactivated because it's wonky in WebKit
```

Comment:
If deactivating it still looks good, is there a point in keeping it?

Reply:
Having the link stretch over the entire width isn't ideal because then it expands into the blank space where there's no text or icon. This is not necessarily *wrong* and we could just assume this as expected. But I'd rather have it fit more snugly.

![grafik](https://github.com/crystal-lang/crystal-website/assets/466378/b8e2e165-64f3-4b9f-8003-ba725eb95682)